lib/slayer/form.rb

```diff
@@ -1,17 +1,31 @@
+require 'virtus'
+require 'dry-validation'
+
 module Slayer
   class Form
-    extend ModelAttribute
-    include ActiveModel::Validations if defined?(Rails)
+    attr_reader :errors
+    include Virtus.model
 
-    # Internal: Form Class Methods
     class << self
```

Comment:
I used `def self.foo` everywhere in the service stuff. Do we prefer `class << self` for this codebase?

If so, I'll go change that.